homeassistant/components/camera/__init__.py

```diff
@@ -160,7 +159,6 @@ class Image:
     content: bytes = attr.ib()
 
 
-@bind_hass
```

Comment:
Removing this is potentially breaking for custom integrations. Should we make a dev blog saying we're removing this feature completely and integrations need to adjust if they use it?

Reply:
Yes, I was thinking about it yesterday too, but it was too late for me (terms of time :D) to prepare anything. Will create a dev blog later, should we use the default deprecation period of 6 months?

Reply:
That's ok although I think we can make it shorter too, like 3 months. I don't think this is used a lot.

Reply:
Will change this in dev blog then